src/worker.cpp

```diff
@@ -374,7 +374,7 @@ void Worker::getSystemJson(std::string& systemJson,
 
             if (itrToHW != hwVersionList.end())
             {
-                systemJson += imKwdValue + "_" + (*itrToHW).second + ".json";
+                systemJson += imKwdValue + ".json";
```

Comment:
A) This is a different defect. Push in a different commit.
B) second part is needed. Check for empty string and only then skip.

Reply:
updated